test/test_package.py

```diff
@@ -381,6 +381,16 @@ def test_parse_package_invalid(self):
         filename = os.path.join(test_data_dir, 'invalid_package.xml')
         self.assertRaises(InvalidPackage, parse_package, filename)
 
+    def test_parse_package_xhtml_description(self):
+        filename = os.path.join(test_data_dir, 'xhtml_description.xml')
+        package = parse_package(filename)
+        assert package.description
+        assert package.plaintext_description() == """
+A package with an XHTML description.
+
+This package contains several xhtml tags which are, according to REP-149, meant to be handled properly but "XML tags and multiple whitespaces" may be stripped in some situations.
+        """
```

Comment:
I'm looking for pre-review feedback. Do folks generally agree that this is an appropriate plaintext rendering of the html description in [`test/data/package/xhtml_description.xml`](https://github.com/ros-infrastructure/catkin_pkg/blob/51ee3aad4b7169f04cbf32d78fb7c02066f23d6f/test/data/package/xhtml_description.xml)?

Reply:
I believe that everything inside the """ is getting captured, so we shouldn't expect to see the leading or trailing whitespace and line breaks.

I noticed this as well and pushed 4bba520.  Thanks for the close read!

Reply:
> What you have looks right to me. Please include an additional sentence after the last one to explicitly show that behavior as well.

[`a79c056` (#329)](https://github.com/ros-infrastructure/catkin_pkg/pull/329/commits/a79c0564ce67285821e5d8145423d8638bb03122)